rosa_hcp/rosa-hcp-creating-cluster-with-aws-kms-key.adoc

```diff
@@ -0,0 +1,82 @@
+:_content-type: ASSEMBLY
+[id="rosa-hcp-creating-cluster-with-aws-kms-key"]
+= Creating ROSA with HCP clusters using a custom AWS KMS encryption key
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-creating-cluster-with-aws-kms-key
+
+toc::[]
+
+Create a {product-title} (ROSA) with a {hcp} (HCP) cluster  using a custom AWS Key Management Service (KMS) key.
+
+//include::modules/rosa-sts-creating-a-cluster-quickly-ocm.adoc[leveloffset=+1]
+//include::modules/rosa-sts-associating-your-aws-account.adoc[leveloffset=+2]
+
+[id="rosa-hcp-creating-cluster-with-aws-kms-key-prereqs"]
+== {hcp-title} Prerequisites
+
+To create a {hcp-title} cluster, you must have the following items:
+
+* A configured virtual private cloud (VPC)
+* Account-wide roles
+* An OIDC configuration
+* Operator roles
+
+[id="rosa-hcp-creating-cluster-with-aws-kms-key-creating-vpc"]
+=== Creating a Virtual Private Cloud for your {hcp-title} clusters
+
+You must have a Virtual Private Cloud (VPC) to create {hcp-title} cluster. You can use the following methods to create a VPC: 
+
+* Create a VPC by using a Terraform template
+* Manually create the VPC resources in the AWS console
+
+[NOTE]
+====
+The Terraform instructions are for testing and demonstration purposes. Your own installation requires some modifications to the VPC for your own use. You should also ensure that when you use this Terraform script it is in the same region that you intend to install your cluster. In these examples, use `us-east-2`.
+====
+
+[discrete]
+include::modules/rosa-hcp-vpc-terraform.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See the link:https://github.com/openshift-cs/terraform-vpc-example[Terraform VPC] repository for a detailed list of all options available when customizing the VPC for your needs.
+
+[discrete]
+include::modules/rosa-hcp-vpc-manual.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-getting-started.html[Get Started with Amazon VPC]
+* link:https://developer.hashicorp.com/terraform[HashiCorp Terraform documentation]
+
+include::modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc[leveloffset=+2]
+
+include::modules/rosa-sts-byo-oidc.adoc[leveloffset=+2]
+
+include::modules/rosa-operator-config.adoc[leveloffset=+2]
```

Comment:
In this module "modules/rosa-operator-config.adoc". It would be better to remove **$** before  "arn:aws:iam::${AWS_ACCOUNT_ID}:role/${ACCOUNT_ROLES_PREFIX}-HCP-ROSA-Installer-Role"
rosa create operator-roles --hosted-cp --prefix $OPERATOR_ROLES_PREFIX --oidc-config-id $OIDC_ID --installer-role-arn **$**arn:aws:iam::${AWS_ACCOUNT_ID}:role/${ACCOUNT_ROLES_PREFIX}-HCP-ROSA-Installer-Role

Reply:
removed, thanks